src/modules/iKinGazeCtrl/include/iCub/utils.h

```diff
@@ -150,10 +154,14 @@ class ExchangeData
     bool            saccadesOn;
     bool            neckPosCtrlOn;
     bool            stabilizationOn;
+    bool            useMASClient;
```

Comment:
I'd personally avoid handling this option.

My preference goes to testing this branch carefully - especially in terms of stabilization performance that are based on the gyro readouts - and then merge it. Thus, no switch, no command-line option to manage two different modalities.

Reply:
The only problem here is to update all the conf files of the robots in order to run the MAS server that wraps the imu. It is absolutely doable but it has to be taken in account

Reply:
We don't need to deal with other robots since we'll be testing the branch with ours and eventually merge it. Let's remove the new parameter, then.

Reply:
Sorry @Nicogene I didn't catch the point that not all the IMU's expose the new API.
I would then stop this development until we'll get all the low level layers aligned.